tests/foreman/api/test_contentmanagement.py

```diff
@@ -1432,3 +1432,32 @@ def test_positive_sync_file_repo(
             sat_file = md5_by_url(f'{sat_repo_url}{file}')
             caps_file = md5_by_url(f'{caps_repo_url}{file}')
             assert sat_file == caps_file
+
+    @pytest.mark.tier3
+    def test_positive_allow_reregistration_when_dmi_uuid_changed(
+        self, module_org, module_ak_with_cv, rhel7_contenthost, default_sat
+    ):
+        """Register a content host with a custom DMI UUI, unregistering it, change
+        the DMI UUI, and reregistering it again
+
+        :id: 7f431cb2-5a63-41f7-a27f-62b86328b50d
+
+        :expectedresults: The content registers successfully
+
+        :BZ: 1747177
+
+        :CaseLevel: Integration
+        """
+        rhel7_contenthost.install_katello_ca(default_sat)
+        default_sat.execute(
+            'echo \'{{"dmi.system.uuid": "this-is-made-up"}}\' > /etc/rhsm/facts/uuid.facts'
+        )
+        result = rhel7_contenthost.register_contenthost(module_org.label, module_ak_with_cv.name)
+        assert result.status == 1
+        result = rhel7_contenthost.unregister()
+        assert result.status == 0
+        default_sat.execute(
+            'echo \'{{"dmi.system.uuid": "another-made-up"}}\' > /etc/rhsm/facts/uuid.facts'
+        )
+        result = rhel7_contenthost.register_contenthost(module_org.label, module_ak_with_cv.name)
+        assert result.status == 1
```

Comment:
as you mention in the expected results host should be successfully registered, shouldn't this be` assert result.status == 0`?

Reply:
@Gauravtalreja1 The results as 1.  I'll double check again and paste the results here for you.

Reply:
I have changed the registration so it outputs a `status==0` for success.  I'm not sure why the previous one gives it a status of 1 and being successful.